lib/cyclonedx/report.rb

```diff
@@ -9,13 +11,16 @@
 
 module Cyclonedx
   class Report
+    DEFAULT_COMPONENT_TYPE = "application".freeze
+    class CycloneDXInvalidFormatError < StandardError; end
+
     def initialize(scan_reports, config = {})
       @scan_reports = scan_reports
       @config = config
     end
 
     CYCLONEDX_SPEC_VERSION = "1.3".freeze
-    CYCLONEDX_VERSION = "1".freeze
+    CYCLONEDX_VERSION = 1
```

Comment:
Why was the `freeze` taken out?

Reply:
Rubocop said it was redundant `Style/RedundantFreeze: Do not freeze immutable objects, as freezing them has no effect.`